### PR TITLE
fix(otelcol/logs): avoid sending metadata attributes in json body

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Fill in your description here.

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -2769,7 +2769,38 @@ otelcol:
...
+              ## Required to be used in headers
+              - _sourceCategory
+              - _sourceHost
+              - _sourceName
+              ## Attributes to be send as fields
+              - _collector
+              - k8s.*
+              - pod_.*
+              - namespace_.*
+              ## ToDo: fix below duplication after https://github.com/SumoLogic/sumologic-otel-collector/pull/245 being resolved
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:2790]: By `fix` we mean .... remove all below listed entries/regexes?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:2790]: yes

Comment (pmalek-sumo) [reply deploy/helm/sumologic/values.yaml:2790]: So.... Can we change the comment to say so :) ?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:2790]: hmm..., maybe

**Review by pmalek-sumo — APPROVED** [review]

**Comment by pmalek-sumo** [comment]
> avoid sending metadata attributes for json format

Shouldn't this be 

> avoid sending metadata attributes in json body

(or similar) ?

**Pull request merged**